src/main/java/org/openstreetmap/atlas/geography/atlas/raw/slicing/RawAtlasSlicer.java

```diff
@@ -1319,7 +1319,9 @@ private void sliceArea(final Area area)
             final Set<Long> relationIds = new HashSet<>();
             area.relations().forEach(relation -> relationIds.add(relation.getIdentifier()));
             final CompleteLine lineFromArea = new CompleteLine(area.getIdentifier(),
-                    area.asPolygon(), area.getTags(), relationIds);
+                    JTS_POLYLINE_CONVERTER.backwardConvert(
```

Comment:
Is it worth adding a test case for this edge case?

Reply:
probably 💀for now I'm prioritizing a fix since this is a one-in-a-million corner case, but I'll see if I can capture the behavior in a sufficient unit test